fibdrv.c

```diff
@@ -20,9 +20,10 @@ MODULE_VERSION("0.1");
 #define MAX_LENGTH 92
 
 static dev_t fib_dev = 0;
-static struct cdev *fib_cdev;
 static struct class *fib_class;
 static DEFINE_MUTEX(fib_mutex);
+static int major = 0;
```

Comment:
Combine the declaration of major and minor in a single line. i.e.,
```c
static int major = 0, minor = 0;
```

Reply:
Ok, you're right, thanks for the correction.